From d4b681e1ddf95875e0fc0ee44fa8a2cf1b0442bf Mon Sep 17 00:00:00 2001
From: Andreas Schwab <schwab@linux-m68k.org>
Date: Wed, 3 Jul 2013 12:48:59 +0200
Subject: [PATCH] * doc/c-mips.texi (MIPS Opts): Fix use of @itemx.  *
 doc/c-score.texi (SCORE-Opts): Likewise.  * doc/c-tic54x.texi
 (TIC54X-Directives): Likewise.

---
 gas/doc/c-mips.texi   |  4 ++--
 gas/doc/c-score.texi  | 10 +++++-----
 gas/doc/c-tic54x.texi |  2 +-
 3 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/gas/doc/c-mips.texi b/gas/doc/c-mips.texi
index d3860f1..50b54df 100644
--- a/gas/doc/c-mips.texi
+++ b/gas/doc/c-mips.texi
@@ -1,5 +1,5 @@
 @c Copyright 1991, 1992, 1993, 1994, 1995, 1997, 1999, 2000, 2001,
-@c 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009, 2010, 2011
+@c 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009, 2010, 2011, 2013
 @c Free Software Foundation, Inc.
 @c This is part of the GAS manual.
 @c For copying conditions, see the file as.texinfo.
@@ -234,7 +234,7 @@ the @samp{mad} and @samp{madu} instruction, and to not schedule @samp{nop}
 instructions around accesses to the @samp{HI} and @samp{LO} registers.
 @samp{-no-m4650} turns off this option.
 
-@itemx -m3900
+@item -m3900
 @itemx -no-m3900
 @itemx -m4100
 @itemx -no-m4100
diff --git a/gas/doc/c-score.texi b/gas/doc/c-score.texi
index 3af20a3..da1391b 100644
--- a/gas/doc/c-score.texi
+++ b/gas/doc/c-score.texi
@@ -1,4 +1,4 @@
-@c Copyright 2009, 2011
+@c Copyright 2009, 2011, 2013
 @c Free Software Foundation, Inc.
 @c This is part of the GAS manual.
 @c For copying conditions, see the file as.texinfo.
@@ -37,7 +37,7 @@ implicitly with the @code{gp} register. The default value is 8.
 @item -EB
 Assemble code for a big-endian cpu
 
-@itemx -EL
+@item -EL
 Assemble code for a little-endian cpu
 
 @item -FIXDD 
@@ -49,13 +49,13 @@ Assemble code for no warning message for fix data dependency
 @item -SCORE5
 Assemble code for target is SCORE5
 
-@itemx -SCORE5U
+@item -SCORE5U
 Assemble code for target is SCORE5U
 
-@itemx -SCORE7
+@item -SCORE7
 Assemble code for target is SCORE7, this is default setting
 
-@itemx -SCORE3
+@item -SCORE3
 Assemble code for target is SCORE3
 
 @item -march=score7
diff --git a/gas/doc/c-tic54x.texi b/gas/doc/c-tic54x.texi
index 75398da..ab54896 100644
--- a/gas/doc/c-tic54x.texi
+++ b/gas/doc/c-tic54x.texi
@@ -345,7 +345,7 @@ Assign @var{name} the string @var{string}.  String replacement is
 performed on @var{string} before assignment.
 
 @cindex @code{eval} directive, TIC54X
-@itemx .eval @var{string}, @var{name}
+@item .eval @var{string}, @var{name}
 Evaluate the contents of string @var{string} and assign the result as a
 string to the subsym @var{name}.  String replacement is performed on
 @var{string} before assignment. 
-- 
1.8.2.1

